bitnami/redis/templates/metrics-svc.yaml

```diff
@@ -17,10 +17,9 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.metrics.service.type }}
-  {{ if eq .Values.metrics.service.type "LoadBalancer" -}} {{ if .Values.metrics.service.loadBalancerIP }}
+  {{ if and (eq .Values.metrics.service.type "LoadBalancer") .Values.metrics.service.loadBalancerIP }}
   loadBalancerIP: {{ .Values.metrics.service.loadBalancerIP }}
-  {{ end -}}
-  {{- end -}}
+  {{- end }}
```

Comment:
You should also apply this changes at `bitnami/redis-cluster/templates/metrics-svc.yaml`, right?

Reply:
Right - I've applied them in this PR and have opened this PR is response to this comment: https://github.com/bitnami/charts/pull/2334#issuecomment-614728542

Thanks for bearing with me on these PRs. Should I add these changes to that PR?